src/components/CommentUsername/CommentUsername.js

```diff
@@ -10,7 +10,7 @@ const styles = StyleSheet.create({
   username: {
     fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(12),
-    color: COLORS.blackgrey,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the username has been changed from `COLORS.blackgrey` to `COLORS.white2`. Please ensure that this color change is consistent with the overall theme and visibility on all screens where this component is used.
```diff
-    color: COLORS.blackgrey,
+    color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->